packages/esm-appointments-app/src/appointments-calendar/daily-calendar-view.component.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import dayjs from 'dayjs';
+import isBetween from 'dayjs/plugin/isBetween';
+import styles from './daily-calendar.scss';
+import { CalendarType } from '../types';
+import { dailyHours } from '../helpers';
+import DailyHeader from './daily-header.component';
+import DailyWorkloadView from './daily-view-workload.component';
+dayjs.extend(isBetween);
+function DailyCalendarView({
+  type = 'daily',
+  events,
+}: {
+  type: CalendarType;
+  events: { appointmentDate: string; service: Array<any>; [key: string]: any }[];
```

Comment:
This can be extracted to `DailyCalendarViewProps` to make it more readable, also avoid using `Array<any>` provide the correct type for it.